packages/braid-design-system/src/lib/components/Hidden/Hidden.tsx

```diff
@@ -32,18 +39,26 @@ export const Hidden = ({
   data,
   ...restProps
 }: HiddenProps) => {
-  if (process.env.NODE_ENV === 'development' && screen) {
-    // eslint-disable-next-line no-console
-    console.warn(
-      `You used the "screen" prop on Hidden, but this probably doesn't do what you expect. If you're trying to provide content to screen readers without rendering it to the screen, use the <HiddenVisually> component instead. The "screen" prop is likely to be deprecated in a future release.`,
-    );
+  if (process.env.NODE_ENV === 'development') {
+    if (typeof screen !== 'undefined') {
+      // eslint-disable-next-line no-console
+      console.warn(
+        dedent`
+        The "screen" prop is deprecated and will be removed in a future release.
+        For content designed to improve the screen reader experience, please use use the <HiddenVisually> component instead.
+        Alternatively, for content designed to be hidden unconditionally, remove the "screen" prop.`,
+      );
+    }
   }
 
   const inText = Boolean(useContext(TextContext));
   const inHeading = Boolean(useContext(HeadingContext));
 
   const hiddenOnScreen = Boolean(screen);
   const hiddenOnPrint = Boolean(print);
+
+  const hiddenAlways = hiddenOnScreen || (!above && !below && !print);
```

Comment:
Think we need a changeset to capture the breaking change here — hiding when no props are specified.